client/elife-theme/src/elements/ui/Action.js

```diff
@@ -0,0 +1,9 @@
+import { css } from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+export default css`
+  // Action is sometimes a Link underneath & sometimes a Button
+  // eLife styling makes a button's font: Noto Sans SemiBold
+  // This reverses that override
+  font-family: ${th('fontInterface')};
```

Comment:
Hmm, also this overriding the override isn't working. I guess because the Button one is more specific.

So now we have `Action` text (Pre-fill my details) which is more bold than it should be (because it's actually a `Button` with different styling):
![image](https://user-images.githubusercontent.com/15656538/47922257-86ad9a00-deae-11e8-9376-f33feb96a489.png)

:thinking:

Reply:
Fixed in the latest commit

Reply:
I think we need option 3. I might now know enough about the implications for pubsweet, but my understanding is if we need something, we do it, and if it's helpful to pubsweet we add it.

I don't think pubsweet should prevent us doing what we need to do in this instance, and it seems like it might helpful to others to have the option to have different fonts/weights on "buttons" and "links". So unless there's a problem or reason I'm not aware of I think we should we should go with:

> adding a prop to pubsweet's Action component, to provide us with some way of differentiating between Actions which are actually Links & Actions which are actually Buttons

One thing that might be worth noting is that we have `Link`s that use both Noto Sans and Noto Serif fonts - not sure if its related but surely using different `font-family`s